addons/account/models/account_move.py

```diff
@@ -1238,25 +1238,28 @@ def _validate_move_modification(self):
 
     @api.constrains('name', 'journal_id')
     def _check_unique_sequence_number(self):
-        if not self:
+        moves = self.filtered(lambda move: move.state == 'posted')
+        if not moves:
+            return
+
+        where_params = []
+        for move in moves:
+            where_params += [move.id, move.name, move.state, move.type, move.journal_id.id]
+
+        if not where_params:
             return
 
+        values_table = ','.join('(%s, %s, %s, %s, %s)' for move in moves)
         self._cr.execute('''
-            WITH current_moves (id, name, state, company_id, journal_id, type) as ( values {} )
-            SELECT move2.id
-            FROM (    SELECT id, name, state, company_id, journal_id, type from account_move
-            UNION ALL SELECT id, name, state, company_id, journal_id, type from current_moves
-            ) move
+            SELECT move.id
+            FROM (VALUES ''' + values_table + ''') AS move(id, name, state, type, journal_id)
```

Comment:
no way to put this in the where-params?

Reply:
params are already in where_params....

Reply:
No way to put values_table in the 2nd arg of the execute method instead of concatenating strings?